statsmodels/examples/ex_generic_mle_tdist.py

```diff
@@ -156,15 +163,6 @@ def nloglikeobs(self, params):
 # fit_ks works well, but no bse or other result statistics yet
 
 
-
-
-
-#import for kstest based estimation
-#should be replace
-# FIXME: importing these patches scipy distribution classes in-place.
-#  Don't do this.
-import statsmodels.sandbox.distributions.sppatch  # noqa:F401
```

Comment:
don't change this import location
Given that this is monkey patching scipy, the location of the import is important and might change other results.